runtime/include/qio/qio_formatted.h

```diff
@@ -46,12 +46,10 @@
 extern "C" {
 #endif
 
-extern int qio_glocale_utf8; // for testing use.
-#define QIO_GLOCALE_UTF8 1
-#define QIO_GLOCALE_ASCII 2
-#define QIO_GLOCALE_OTHER -1
-
-void qio_set_glocale(void);
+//extern int qio_glocale_utf8; // for testing use.
```

Comment:
Why not just remove these? The SCCS system can resurrect it if we ever need it. In my experience this sort of thing never gets cleaned up and we just end up with a bunch of commented-out code with nobody knowing why it's there.

Reply:
Yes I meant to delete this comment before we got to the PR review but forgot. So thanks for the reminder!